var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -31,6 +31,7 @@ class Pango(AutotoolsPackage):
     depends_on("libxft", when='+X')
     depends_on("glib")
```

Comment:
```suggestion
    depends_on("glib@2.33.12:")
```
Similarly, `pkgconfig` should be `@0.9.0:`

Reply:
@adamjstewart Done. I added a couple other hidden dependencies that I found were checked by `configure` in `config.log` (posted in #12383).